### PR TITLE
Create test branch from release candidate

### PR DESCRIPTION
**Pull request by shammowla**
The production test workflow are using automated tests provided by test branches created from tags. Updated workflow to create a test branch prior to release. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

**Comment by jfkhoury** [comment]
Thank you for the description!

**Review by jfkhoury — APPROVED** [review]
Looks good @shammowla but I will let Jon and Martin verify.

**Review by jonsnyder — CHANGES_REQUESTED** [review]
I see you found a workflow job to create the branch. This could work instead of the javascript changes I suggested, but you need to put it into the deployRelease workflow, and setup the dependencies correctly.

validate -> release -> createTestBranch -> record
________`---------->prerelease -----------/

**Comment by shammowla** [comment]
@jonsnyder I moved the job to the deployRelease workflow.

**Comment by shammowla** [comment]
@jonsnyder made the changes we discussed.

File: .github/workflows/deployRelease.yml [hunk 1]
```diff
@@ -18,6 +18,18 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           version: ${{ github.event.inputs.version }}
+  createTestBranch:
+    name: Create test branch from release tag
+    runs-on: ubuntu-latest
+    needs: validate
```
Comment (jonsnyder) [inline .github/workflows/deployRelease.yml:24]: This should be `needs: release`, and then move this job below release for consistency.

**Review by jonsnyder — CHANGES_REQUESTED** [review]

**Pull request merged**